requirements.txt

```diff
@@ -1,4 +1,6 @@
 pandas>=1.0.0
 geopandas>=0.8.1
 Rtree>=0.9.4
+shapely>=1.7.1
+netCDF4>=1.5.4
```

Comment:
You'll need to update the setup.py file as well : https://github.com/pyronear/pyronear-datascience/blob/master/setup.py#L37 ;)

Reply:
yep